### PR TITLE
Introduce new README.md

### PR DESCRIPTION
**Pull request by mkutlak**
:tm:

Signed-off-by: Martin Kutlak <mkutlak@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,44 @@
...
+Captured errors and exceptions are reported to [ABRT Analytics](https://retrace.fedoraproject.org/faf) without any user interation.
+
+Features
+---
+**Automatic error monitoring and reporting** - Exceptions and errors in Python are automatically captured and reported.  
+
+
+Using sABRTooth
+---
+1) Install the `sABRTooth`.
```
Comment (michalfabik) [inline README.md]: The "the" is superfluous here. (Or did you mean to say "the sABRTooth package"?)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,44 @@
-# abrt-dev-modules
-Developer modules for tracking exceptions in software
+<p align="center">
+<a href="https://retrace.fedoraproject.org/">
+    <img alt="ABRT" src="https://github.com/abrt/abrt/blob/master/icons/hicolor_apps_scalable_abrt.svg" width="123">
```
Comment (ernestask) [inline README.md]:
```suggestion
    <img alt="ABRT" src="https://github.com/abrt/abrt/blob/master/icons/hicolor_apps_scalable_abrt.svg" width="128">
```

Comment (ernestask) [reply README.md]: It could be slightly blurry if it’s not a power-of-two size.

Comment (ernestask) [reply README.md]: (I would commit and squash, but I don’t think it will be possible to selectively squash)

**Comment by ernestask** [comment]
Introducing the latest of readme technology.

**Pull request merged**